internal/kubernetes_test.go

```diff
@@ -349,22 +350,21 @@ func TestKubeInvalidConfig3(t *testing.T) {
 }
 
 func TestKubeInvalidSecretType(t *testing.T) {
-	testRequestKube(t, &Exporter{
-		KubeIncludeNamespaces: []string{"default"},
-		KubeSecretTypes:       []string{"aze"},
-	}, func(m []model.MetricFamily) {
-		metrics := getMetricsForName(m, "x509_read_errors")
-		assert.Equal(t, 1., metrics[0].GetGauge().GetValue())
-	})
+	_, err := ParseSecretType("aze")
```

Comment:
Why did you change the way the test was written? If you want to test in another way, I would prefer that you add a testcase rather than changing the existing when introducing a new feature.

Reply:
I see, indeed it is the way to go :+1: